packetbeat/protos/cassandra/frame.go

```diff
@@ -0,0 +1,900 @@
+/**
+https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec
+
+  The CQL binary protocol is a frame based protocol. Frames are defined as:
+
+      0         8        16        24        32         40
+      +---------+---------+---------+---------+---------+
+      | version |  flags  |      stream       | opcode  |
+      +---------+---------+---------+---------+---------+
+      |                length                 |
+      +---------+---------+---------+---------+
+      |                                       |
+      .            ...  body ...              .
+      .                                       .
+      .                                       .
+      +----------------------------------------
+
+  The protocol is big-endian (network byte order).
+
+  some code derived from https://github.com/gocql/gocql
```

Comment:
This file seems to be mostly a copy of gocql. How about putting the mostly copied code into separate sub-package and export the required symbols?

E.g. `github.com/elastic/beats/packetbeat/protos/cassandra/internal/gocql`.

This package will be used by cassandra parser only + we can add notes including original license and so on.